apps/molgenis-components/src/components/layout/MolgenisMenu.vue

```diff
@@ -73,8 +75,6 @@ export default {
     items: Array,
     /** logo to show*/
     logo: String,
-    /** session information, so we can check role permissions */
```

Comment:
I must say that in this case I like passing props better. We should only use the store if you want to mutate the data. Otherwise please use props.